compiler/rustc_borrowck/locales/en-US.ftl

```diff
@@ -101,12 +89,63 @@ borrowck_capture_mut =
 borrowck_capture_move =
     capture is moved because of use here
 
+borrowck_var_borrow_by_use_place_in_generator =
+    {$is_single_var ->
```

Comment:
Some languages don't just have singular/plural, but also have a [dual](https://en.wikipedia.org/wiki/Dual_(grammatical_number)). The only bigger modern language where I imagine a use would be arabic/maltese, but I don't speak it so I don't know for sure. I think it's still would be better to match on numbers instead of bools here.

Reply:
Matching on numbers requires knowing the number of var, I don't think I can do that for now.

maybe have a look at `rust/compiler/rustc_borrowck/src/diagnostics/conflict_errors.rs:1102`

Reply:
Hmm it's not trivial... fair point. I think eventually it would be nice to support one/two/many instead of one/many, but maybe that can be done at a later point in time. Case rested.